third_party/maya/lib/usdMaya/usdListShadingModes.cpp

```diff
@@ -0,0 +1,89 @@
+//
```

Comment:
usdListShadingModes.cpp was moved to listShadingModesCommand a little while back, so I think this file can just be removed.